README.rst

```diff
@@ -155,24 +155,27 @@ Managers can confirm a Booking using workflow transitions.
 The workflow transition triggers an email to be sent to the booker (see below).
 
 
-Booking event notifications
----------------------------
+Booking Notifications
+---------------------
 
-There are addional automated email be used to notify booking owner when his booking has been created, accepted
-or re-scheduled.
+There are automated notifications implementend by the foollowing behaviors:
 
-Rules are **NOT automatically** enabled in every Booking Folder.
-If you want to send some notification, you only need to enable them from the Booking folder.
+* `redturtle.prenotazioni.behavior.notification_appio` (Notify via AppIO gateway)
+* `redturtle.prenotazioni.behavior.notification_email` (Notify via Email gateway)
+* `redturtle.prenotazioni.behavior.notification_sms` (Notify via SMS gateway)
 
-The rules which are available by default:
+Each behavior is implementing the following notification types:
+* `booking-accepted` (Invia un messaggio all'utente quando la prenotazione è stata accettata)
+* `booking-moved` (Invia un messaggio all'utente quando la data della prenotazione viene cambiata)
+* `booking-created-user` (Invia un email all'utente quando la prenotazione è stata creata)
+* `booking-refuse` (Invia un email all'utente quando la prenotazione è stata rifiutata)
+* `booking-reminder` (Booking reminder message)
 
-* `booking-accepted` (Invia un'email all'utente quando la prenotazione è stata accettata)
-* `booking-moved` (Invia un'email all'utente quando la data della prenotazione viene cambiata)
-* `booking-created-user` (Invia un'email all'utente quando la prenotazione è stata creata)
-* `booking-refuse` (Invia un'email all'utente quando la prenotazione è stata rifiutata)
-* `booking-confirm` (Conferma automatica prenotazioni)
+Notifications are **NOT automatically** enabled in every Booking Folder.
+If you want to send some notification, you only need to enable the by assigning the behavior to PrenotazioniFolder c.t.
 
-You can also use some placeholders that will be replaced with some booking infos. Here is a list:
+You can create your own email templates for the booking events(confirm, refuse, create, delete, reminder).
+The temlates are being saved in the PrenotazioniFolder object.
```

Comment:
Write also some infos about SMS: they are not implemented by default and if someone wants to use them, need to write some code.

Add some snippets on how adapters work and what's the base cose to write to enable SMS functionality. Something like this: https://github.com/RegioneER/rer.sitesearch#register-new-advanced-filters

Reply:
Done